components/Modals/BuyNFT/index.tsx

```diff
@@ -381,233 +400,243 @@ function Component() {
   return (
     <>
       <div className={styles.outerWrapper}>
-        <div
-          className={styles.containerClose}
-          onClick={() => updateModalBuyNft(false)}
-        />
+        <div className={styles.containerClose} onClick={closeModal} />
         <div className={styles.container}>
           <div className={styles.innerContainer}>
             <div className={styles.closeIcon}>
-              <Image
-                alt="close-ico"
-                src={CloseIcon}
-                onClick={() => {
-                  updateModalBuyNft(false);
-                  Router.push("/");
-                }}
-              />
+              <Image alt="close-ico" src={CloseIcon} onClick={closeModal} />
             </div>
-            <div className={styles.contentWrapper}>
-              <h3 className={styles.headerInfo}>buy this nft username</h3>
-              <div className={styles.usernameInfoWrapper}>
-                <h2 className={styles.usernameInfoYellow}>
-                  {user?.name.length! > 20
-                    ? `${user?.name.slice(0, 8)}...${user?.name.slice(-8)}`
-                    : user?.name}
-                </h2>
-                <h2 className={styles.usernameInfo}>is available</h2>
+            {txType === "AddFunds" &&
+            (loginInfo.txStatusLoading === "true" ||
+              loginInfo.txStatusLoading === "submitted") ? (
+              <div className={styles.wrapperLoading}>
+                <div>{spinner}</div>
+                <h4 style={{ marginTop: "2%" }}>
+                  To continue, please wait until the Add Funds transaction gets
+                  finalised on the Zilliqa blockchain
+                </h4>
               </div>
-              {loginInfo.address === null ? (
-                <div
-                  style={{
-                    display: "flex",
-                    width: "100%",
-                    justifyContent: "center",
-                  }}
-                >
-                  <p style={{ marginTop: "1%" }}>To continue:&nbsp;</p>
-                  <button className="button" onClick={handleConnect}>
-                    <p>LOG IN</p>
-                  </button>
+            ) : txType === "AddFunds" ? (
```

Comment:
there is a mistake here/ not needed @DevPH-Wistkey

Reply:
Updated, please kindly check it @tralkan